rust/src/lockfile.rs

```diff
@@ -250,9 +251,9 @@ mod ffi {
         };
 
         for pkg in packages {
-            let name: String = unsafe { from_glib_none(dnf_package_get_name(pkg)) };
-            let evr: String = unsafe { from_glib_none(dnf_package_get_evr(pkg)) };
-            let arch: String = unsafe { from_glib_none(dnf_package_get_arch(pkg)) };
+            let name: Borrowed<GString> = unsafe { from_glib_borrow(dnf_package_get_name(pkg)) };
```

Comment:
I have a feeling we may soon want to start wrapping a `NonNull<DnfPackage>` and wire these getters there.